appengine/flexible/tasks/create_app_engine_queue_task.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import argparse
+import base64
+import datetime
+import pprint
+
+from googleapiclient import discovery
+
+
+def seconds_from_now_to_rfc3339_datetime(seconds):
+    """Return an RFC 3339 datetime string for a number of seconds from now."""
+    d = datetime.datetime.utcnow() + datetime.timedelta(seconds=seconds)
+    return d.isoformat('T') + 'Z'
+
+
+def create_task(project, queue, location, payload=None, in_seconds=None):
+    """Create a task for a given queue with an arbitrary payload."""
+    client = get_client()
+
+    url = '/log_payload'
+    body = {
+        'task': {
+            'app_engine_task_target': {
+                'http_method': 'POST',
+                'relative_url': url
+            }
+        }
+    }
+
+    if payload is not None:
+        # Payload is a string (unicode), so 
+        body['task']['app_engine_task_target']['payload'] = base64.b64encode(
+            payload.encode()).decode()
+
+    if in_seconds is not None:
+        scheduled_time = seconds_from_now_to_rfc3339_datetime(in_seconds)
+        body['task']['schedule_time'] = scheduled_time
+
+    queue_name = 'projects/{}/locations/{}/queues/{}'.format(
+        project, location, queue)
+
+    print('Sending task {}'.format(pprint.pformat(body)))
+
+    response = client.projects().locations().queues().tasks().create(
+        parent=queue_name, body=body).execute()
+
+    # By default CreateTaskRequest.responseView is BASIC, so not all
+    # information is retrieved by default because some data, such as payloads,
+    # might be desirable to return only when needed because of its large size
+    # or because of the sensitivity of data that it contains.
+    print('Created task {}'.format(response['name']))
+    return response
+
+
+def get_client():
+    """Build an http client."""
+    DISCOVERY_URL = 'https://cloudtasks.googleapis.com/$discovery/rest?version=v2beta2'
```

Comment:
it's not a constant, so it doesn't need to be capital.

Reply:
It's not a constant? It's not modified or replaced anywhere, it's just a string literal with a name, right?